flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/AgentsGenerator.java

```diff
@@ -100,14 +104,17 @@ public class AgentsGenerator {
     private final Gson gson;
     private final SemanticModel semanticModel;
     private static final String BALLERINAX = "ballerinax";
-    private static final String AI_AGENT = "ai";
+    private static final String AI = "ai";
```

Comment:
I would use all AI related constants to a separate section (May be not needed here). 

Check this out: https://github.com/ballerina-platform/ballerina-language-server/blob/fb40fcef6d81b10283e0e6169d3fd40e91ac1c93/flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/Constants.java#L35